commit 926d7ea39507
Author: Dan Minor <dminor@mozilla.com>
Date:   Thu Apr 12 11:31:59 2018 -0700

    Bug 1453740 - Allow 1x1 windows in VP8EncoderImpl::InitEncode; r=pehrsons a=jcristau
    
    A minimized window has a resolution of 1x1. Although we removed minimized windows from the list
    of available windows to share, nothing prevents the user from minimizing it during a call. With
    the current code, this will cause InitEncode to fail, resulting in a fatal release assert.
    
    I tested this patch with window sharing on meet.google.com and I was able to minimize and restore
    the window several times without problem. While minimized, the window appears as a black screen
    to the other meeting participants. It renders normally when restored.
    
    MozReview-Commit-ID: LE2NBiEy8nw
    
    --HG--
    extra : source : 1cece7d5df170946f822705ac7b28e2e57c0fd00
---
 media/webrtc/trunk/webrtc/modules/video_coding/codecs/vp8/vp8_impl.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git media/webrtc/trunk/webrtc/modules/video_coding/codecs/vp8/vp8_impl.cc media/webrtc/trunk/webrtc/modules/video_coding/codecs/vp8/vp8_impl.cc
index 36f8552ad8d1..0fe2c6953868 100644
--- media/webrtc/trunk/webrtc/modules/video_coding/codecs/vp8/vp8_impl.cc
+++ media/webrtc/trunk/webrtc/modules/video_coding/codecs/vp8/vp8_impl.cc
@@ -300,7 +300,7 @@ int VP8EncoderImpl::InitEncode(const VideoCodec* inst,
   if (inst->maxBitrate > 0 && inst->startBitrate > inst->maxBitrate) {
     return WEBRTC_VIDEO_CODEC_ERR_PARAMETER;
   }
-  if (inst->width <= 1 || inst->height <= 1) {
+  if (inst->width < 1 || inst->height < 1) {
     return WEBRTC_VIDEO_CODEC_ERR_PARAMETER;
   }
   if (number_of_cores < 1) {
